src/preload-hover.js

```diff
@@ -4,7 +4,8 @@ export default class PreloadHover {
   constructor(configuration = null) {
     const defaultConfiguration = {
       defaultDomScope: [document.body],
-      debounceTime: 50
+      debounceTime: 50,
+      linkType: true, 
```

Comment:
Could you make `linkType` a string defaulting to `local`? Being a boolean I wouldn't know what `true`/`false` means.

It should accept `local` and `external` and throw if one of those expected values is not provided.